### ISSUE TITLE
Swim bug

### ISSUE THREAD

**Issue by MrBons4i** [tiltedphoques/TiltedEvolution#676]
## Version of Skyrim Together

v1.6.5

## Description

I don't know how, but as soon as I use the mod I can no longer swim. If I go into deep water I just keep running.

## Steps to reproduce

1. Load the mod and start the Game
2. Start a new Game
3. Make Helgen intro
4. Go to a See and now i can´t swimming

## Reproduction rate
100% by me an a Frinde

## Expected result

maby a hotfix

**Comment by Dragonisser** [comment]
![image](https://github.com/tiltedphoques/TiltedEvolution/assets/4233773/e8157f44-d86e-420c-a64f-b33743763d8b)

**Comment by rfortier** [comment]
All I had to do was jump in a lake, right?

I tried it on my 1.6.5 build of /master, and had the same sink-to-bottom. Cosi mentioned his local build worked, but my local build shows the problem. After switching to the /master branch, I did a pretty hard clean, build solution clean, deleting /build, /dist, /vsxmake, Code/sky_ui/build, stuff like that. Just short of wipe all and pull from git again.

I'll try a couple other variations.
